### PR TITLE
Update indian-journal-of-medical-research.csl

### PR DESCRIPTION
**Pull request by POBrien333**
Via https://forums.zotero.org/discussion/72668/indian-journal-of-medical-research-reference-style
Guidelines not updates (style is years old). Style based on recent papers.

### CODE REVIEW

File: indian-journal-of-medical-research.csl [hunk 1]
```diff
@@ -237,8 +213,8 @@
     <sort>
       <key variable="citation-number"/>
     </sort>
-    <layout prefix="(" suffix=")" delimiter=",">
-      <text variable="citation-number"/>
+    <layout vertical-align="sup" delimiter=",">
```
Comment (adam3smith) [inline indian-journal-of-medical-research.csl]: what's going on here? Are these superscript or not? Presumably yes, right? If so, pls. remove the `vertical-align="baseline"` below - thanks!

Comment (rmzelle) [reply indian-journal-of-medical-research.csl]: Looks like this should be superscript, yes (checked in https://doi.org/10.4103/ijmr.IJMR_616_18). I'll go ahead and make the change and merge.

**Comment by rmzelle** [comment]
Thanks!

**Comment by POBrien333** [comment]
Thanks for the fix.

**Pull request merged**